django/core/management/base.py

```diff
@@ -240,6 +241,14 @@ def __init__(self, stdout=None, stderr=None, no_color=False, force_color=False):
             self.style = color_style(force_color)
             self.stderr.style_func = self.style.ERROR
 
+        if self.requires_system_checks in [False, True]:
+            warnings.warn(
+                "Using 'requires_system_checks = %r' is deprecated, "
+                "use 'requires_system_checks = %r' instead." %
+                (self.requires_system_checks, '__all__' if self.requires_system_checks else []),
+                RemovedInDjango40Warning,
+            )
```

Comment:
We should update `self.requires_system_checks` if it's a boolean, e.g.
```python
self.requires_system_checks = '__all__' if self.requires_system_checks else []
```
we can also add a module level constant:
```
ALL_CHECKS = '__all__'
```

Reply:
Updated!